Tests/Unit/MiniAppPreferencesTests.swift

```diff
@@ -0,0 +1,29 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+class MiniAppPreferencesTests: QuickSpec {
+
+    override func spec() {
```

Comment:
I think it's not necessary in this case, but should we mock Userdefaults contained in MiniAppPreferences for unit testing by using UserDefaults().removePersistentDomain(forName: "com.rakuten.tech.mobile.miniapp")